File: atomichunter-rest/src/main/java/net/vniia/skittles/dto/VacancyCompetenceScoreRequestDto.java
package net.vniia.skittles.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import net.vniia.skittles.entities.VacancyCompetenceScore;

import java.util.List;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class VacancyCompetenceScoreRequestDto {
    private Long vacancyRespondId;
    private Long interviewId;
    private EmployeeDto employee;
    private List<CompetenceWeightScoreDto> competenceWeightScoreList;
    private String comment;
}

File: atomichunter-web/src/app/dto/VacancyCompetenceScoreRequestDto.ts
import {Employee} from "./Employee";
import {CompetenceWeightScore} from "./CompetenceWeightScore";

export class VacancyCompetenceScoreRequestDto {
    interviewId: number;
    vacancyRespondId: number;
    employee: Employee;
    competenceWeightScoreList: CompetenceWeightScore[];
    comment: string;
    vacancyCompetenceId: number;
}